templates/import-meetup-events-settings.php

```diff
@@ -192,6 +192,31 @@
                         </td>
                     </tr>
 
+                    <tr>
+                        <th scope="row">
+                            <?php _e( "Don't Update these data.", "import-meetup-events" ); ?> : 
+                        </th>
+                        <td>
+                            <?php
+                            $meetup_options = isset($meetup_options['dont_update'])? $meetup_options['dont_update'] : array();
+                            $sdontupdate = isset( $meetup_options['status'] ) ? $meetup_options['status'] : 'no';
+                            $cdontupdate = isset( $meetup_options['category'] ) ? $meetup_options['category'] : 'no';
+                            ?>
+                            <input type="checkbox" name="meetup[dont_update][status]" value="yes" <?php if( $sdontupdate == 'yes' ) { echo 'checked="checked"'; } if (!ime_is_pro()) {echo 'disabled="disabled"'; }?> />
```

Comment:
Use `checked()` and `disabled()` function of wp here. 
https://developer.wordpress.org/reference/functions/checked/
https://developer.wordpress.org/reference/functions/disabled/